src/main/java/com/kjh/ticketreserve/controller/MovieController.java

```diff
@@ -0,0 +1,68 @@
+package com.kjh.ticketreserve.controller;
+
+import com.kjh.ticketreserve.MovieRequest;
+import com.kjh.ticketreserve.MovieResponse;
+import com.kjh.ticketreserve.exception.NotFoundException;
+import com.kjh.ticketreserve.jpa.MovieRepository;
+import com.kjh.ticketreserve.model.Movie;
+import com.kjh.ticketreserve.service.MovieService;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+@RestController
+public class MovieController {
+
+    private final MovieRepository movieRepository;
+    private final MovieService movieService;
+
+    public MovieController(MovieRepository movieRepository, MovieService movieService) {
+        this.movieRepository = movieRepository;
+        this.movieService = movieService;
+    }
+
+    @PostMapping("/admin/movie")
```

Comment:
많이 사용되는 HTTP API 경로 관습은 엔터티 이름의 복수를 사용합니다. 예를 들어 이 API의 경우 `/admin/movie`보다 `/admin/movies`를 일반적으로 선호합니다.

```suggestion
    @PostMapping("/admin/movies")
```

Reply:
엔터티의 복수형으로 사용하는게 적합한 것 같아서 다음 커밋에서 수정하도록 하겠습니다!